### ISSUE TITLE
Generated OME-XML in OME-TIFF files is not schema compliant due to out-of-order AcquisitionDate element

### ISSUE THREAD

**Issue by jmuhlich** [BodenmillerGroup/imctools#114]
* imctools version: 2.1.7
* Python version: any
* Operating System: any

### Description

The XML generated by `imctools.io.utils.get_ome_xml` puts the `AcquisitionDate` element after the `Pixels` element inside the `Image` element, but the official OME XML Schema requires that `AcquisitionDate` comes first. BioFormats (and likely other tools) will actually handle this OK since it doesn't formally validate the XML, but some tools do expect schema-compliant XML.

This should be easy to fix by moving the `AcquisitionDate` to be the first child under `image_element` here:
https://github.com/BodenmillerGroup/imctools/blob/v2.1.7/imctools/io/utils.py#L88

OME XML Schema docs for Image to show the required element ordering:
https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome_xsd.html#Image

### What I Did

```
$ .../bioformats/tools/xmlvalid example.ome.tiff
Parsing schema path
http://www.openmicroscopy.org/Schemas/OME/2016-06/ome.xsd
Validating example.ome.tiff
cvc-complex-type.2.4.a: Invalid content was found starting with element 'AcquisitionDate'. One of '{
"http://www.openmicroscopy.org/Schemas/OME/2016-06":ROIRef,
"http://www.openmicroscopy.org/Schemas/OME/2016-06":MicrobeamManipulationRef,
"http://www.openmicroscopy.org/Schemas/OME/2016-06":AnnotationRef
}' is expected.
Error validating document: 1 errors found
```

**Comment by jwindhager** [comment]
Hi @jmuhlich, thanks for reporting the issue! I drafted a pull request (see above), but need to wait for the authors/maintainers to polish & release it. I'll keep you posted!

**Comment by jmuhlich** [comment]
Thanks, that change looks good to me.

**Comment by plankter** [comment]
Hi @jmuhlich,

The issue should be fixed in the version 2.1.8.

@jwindhager, thank you for the pull request!
